website/src/docs/aws-s3-multipart.md

```diff
@@ -124,27 +127,37 @@ An example of what the return value should look like:
 {
   "presignedUrls": {
     "1": "https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=1&...",
-    "2": "https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=2&...",
-    "3": "https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=3&..."
   },
   "headers": { 
     "1": { "Content-MD5": "foo" },
-    "2": { "Content-MD5": "bar" },
-    "3": { "Content-MD5": "baz" }
   }
 }
 ```
 
-If an error occured, reject the `Promise` with an `Object` with the following keys:
+### `signPart(file, partData)`
```

Comment:
See other comment

Reply:
Do you mean we should not document `signPart`?

Reply:
https://github.com/transloadit/uppy/pull/4187#discussion_r1016929583

I think we should not make this API publicly known because then we are stuck with it. This is a quickfix, not a rethink of the API. That comes in the next stage.